### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Aaahh**
Added tambry's edits too (https://github.com/RPCS3/rpcs3/pull/1210)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,10 @@ If you want to contribute please take a took at the [Coding Style](https://githu
 ### Dependencies
 
 __Windows__
+* [Visual Studios 2015](https://www.visualstudio.com/en-us/downloads/download-visual-studio-vs.aspx)
```
Comment (tambry) [inline README.md]: Should be "Visual Studio 2015" instead.

Comment (Aaahh) [inline README.md]: Yes it should

**Comment by tambry** [comment]
@Aaahh Will you be fixing the small mistake I pointed? Also you need to rebase, so it would be mergable.

**Comment by Aaahh** [comment]
Sure think, I'll fix it later today

**Pull request closed without merging**